client/webserver/site/src/js/markets.ts

```diff
@@ -415,6 +415,9 @@ export default class MarketsPage extends BasePage {
     bind(page.mktBuyField, 'keyup', () => { this.marketBuyChanged() })
     bind(page.rateField, 'change', () => { this.rateFieldChanged() })
     bind(page.rateField, 'keyup', () => { this.previewQuoteAmt(true) })
+    // Verify order view.
+    bind(page.vPass, 'input', () => { this.vPassInputHandler() })
+    bind(page.vUnlockPass, 'input', () => { this.vUnlockPassInputHandler() })
```

Comment:
Kinda unrelated to this PR, but we discussed a similar issue (about hiding error) in another one so I decided to fix it as well while I'm here.